Subject: Correct value of 'include' in ftplugin/python.vim
 'include' needs to be anchored to the beginning of the line as well as
 correctly escaping "\s" to prevent incorrect matches.
Forwarded: 2008-07-16
Author: Jan Hudec <bulb@ucw.cz>
Bug-Debian: http://bugs.debian.org/367259

Index: b/runtime/ftplugin/python.vim
===================================================================
--- a/runtime/ftplugin/python.vim
+++ b/runtime/ftplugin/python.vim
@@ -8,7 +8,7 @@
 
 setlocal cinkeys-=0#
 setlocal indentkeys-=0#
-setlocal include=\s*\\(from\\\|import\\)
+setlocal include=^\\s*\\(from\\\|import\\)
 setlocal includeexpr=substitute(v:fname,'\\.','/','g')
 setlocal suffixesadd=.py
 setlocal comments-=:%
